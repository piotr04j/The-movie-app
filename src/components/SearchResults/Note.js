import React from 'react'

const note = () => {

    return (
        <div className="container my-5">
            <p className="font-weight-bold">Too many search results to display. Refine your search criteria and try again.</p>
            <p> For example, if you have searched "Start Wars". You should try "Star Wars: The Last Jedi".</p>
        </div>
    )
}

export default note;

