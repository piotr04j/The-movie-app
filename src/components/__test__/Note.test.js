import React from 'react';
import renderer from 'react-test-renderer';
import Note from '../SearchResults/Note';

it('displays Note correctly', () =>{
    const tree = renderer.create(<Note/>).toJSON();
    expect(tree).toMatchSnapshot();
});