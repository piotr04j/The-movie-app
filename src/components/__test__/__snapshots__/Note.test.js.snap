// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays Note correctly 1`] = `
<div
  className="container my-5"
>
  <p
    className="font-weight-bold"
  >
    Too many search results to display. Refine your search criteria and try again.
  </p>
  <p>
     For example, if you have searched "Start Wars". You should try "Star Wars: The Last Jedi".
  </p>
</div>
`;
